.eslintrc.js

```diff
@@ -123,6 +123,27 @@ module.exports = {
 				],
 			},
 		},
+
+		{
+			files: ['apps/dotcom/**/*', 'packages/editor/**/*', 'packages/tldraw/**/*'],
+			rules: {
+				'no-restricted-globals': [
```

Comment:
just to confirm - how do these being specified here interact with the ones from above? can we double check that they merge rather than overwrite, and add a comment explaining that that's the case if they do?

Reply:
hah! omg good catch. yes, i had this structured wrong 😬